gluon-mesh-vpn-wireguard-vxlan/files/lib/gluon/upgrade/400-mesh-vpn-wireguard

```diff
@@ -0,0 +1,37 @@
+#!/bin/sh
+
+rm /etc/config/wireguard
+touch /etc/config/wireguard
+
+uci set wireguard.wireguard=wireguard
+
+
+#Ist multidomainsupport aktiviert?
+domain=$(uci get gluon.core.domain)
+if [ -z "$domain" ]; then {                     #falls nein
+        datei="/lib/gluon/site.json";
+} else {                                        #falls ja
+        datei="/lib/gluon/domains/$domain.json";
+}
```

Comment:
This file should be rewritten in Lua and utilize the gluon-site package.

Currently, this fails in constellations where wireguard settings are generated by merging site sections. If this is desired, a check_site script is helpful.